### PR TITLE
AO3-4045 Change error message when chapter character count is too long

### PR DESCRIPTION
**Pull request by pixelred**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4045

## Purpose

makes the chapter 'character limit exceeded' message relevant regardless of whether the user is posting the first chapter of a new work or a new chapter of an existing multi-chapter work.

### CODE REVIEW

**Comment by sarken** [comment]
Whoops, sorry @zz9pzza -- I had pending comments on this. Taking off Ready to Merge label.

File: app/views/chapters/_chapter_form.html.erb [hunk 1]
```diff
@@ -53,13 +53,13 @@
...
   <div class="rtf-html-field">
     <%= f.text_area :content, :id => "content", :class => "mce-editor observe_textlength large" %>
-    <%= live_validation_for_field('content', 
-          :maximum_length => ArchiveConfig.CONTENT_MAX_DISPLAYED, 
-          :minimum_length => ArchiveConfig.CONTENT_MIN, 
-          :tooLongMessage => ts('We salute your ambition! But sadly the content must be less than %{max} characters long. (Maybe you want to create a multi-chaptered work?)', :max => ArchiveConfig.CONTENT_MAX_DISPLAYED.to_s),
+    <%= live_validation_for_field('content',
+          :maximum_length => ArchiveConfig.CONTENT_MAX_DISPLAYED,
+          :minimum_length => ArchiveConfig.CONTENT_MIN,
+          :tooLongMessage => ts('We salute your ambition! But sadly each chapter of a work must be less than %{max} characters long.', :max => ArchiveConfig.CONTENT_MAX_DISPLAYED.to_s),
```
Comment (pixelred) [inline app/views/chapters/_chapter_form.html.erb]: i wanna do this :D

thank you for the pointers! i'll work through these and push the changes ✌️

**Comment by pixelred** [comment]
(hopefully i didn't break anything in the process...)

thank you for explaining all the syntax changes to me. i learned something new :' )

**Comment by sarken** [comment]
Yay, you're welcome! And it all looks good to me. If Travis passes, I will set this to ready to merge!

**Pull request merged**